docs/contribute/cumulative-docs/best-practices.md

```diff
@@ -0,0 +1,310 @@
+
+::::{image} ./images/definition-list-portion-correct.png
+:screenshot:
+:alt: Correctly using inline applies_to in a portion of a definition list item
+::::
+
+
+### Ordered and unordered lists
+
+Reorganize content as needed so the `applies_to` badge is relevant to the entire contents of the list item.
+The recommended placement of the badge varies based on the purpose of the list.
+
+#### If the purpose of the list is to illustrate the difference between situations, put a badge at the start of each item [list-compare-applies_to]
+
+This could mean distinguishing between deployment types, products, lifecycles, or versions.
+Placing the badge at the beginning of the list item, allows the reader to scan the list for the item that is relevant to them.
+
+For example, the [Alerting and action settings in Kibana](https://www.elastic.co/docs/reference/kibana/configuration-reference/alerting-settings) page lists how the default value for the `xpack.actions.preconfigured.<connector-id>.config.defaultModel` setting varies in Serverless/Stack and across versions.
+
+::::{image} ./images/list-correct.png
+:screenshot:
+:alt:
+::::
+
+#### If the list just happens to have one or more items that are only relevant to a specific situation, put the badge at the end of the list item [list-other]
+
+Placing the badge at the end of the list item maintains the flow of the list without distracting the reader with badges while still making it clear that the content in that list item is only applicable to the specified situation.
+
+For example, the [Add filter controls](https://www.elastic.co/docs/explore-analyze/dashboards/add-controls) page lists ways to configure ES|QL controls. Only one of the ways they can be controlled was added for the first time in 9.1.0.
+
+::::{image} ./images/list-misc-correct.png
+:screenshot:
+:alt:
+::::
+
+
+% Reference: Slack conversation
+### Tables
+
+The recommended placement in tables varies based on what part(s) of the table related to the `applies_to` label.
+
+#### If the badge is relevant to the entire row, add the badge to the end of the first column [table-row]
+
+For example, a table that contains one setting per row and a new setting is added in 9.1.0.
+
+<image>
+
+#### If the badge is relevant to one cell or part of a cell, add the badge to the cell it applies to [table-cell]
+
+For example, a new property is available in 9.1.0 for a setting that already existed before 9.1.0.
+
+<image>
+
+#### If the badge is relevant to part of a cell, add the badge to the end of the line it applies to [table-cell-part]
+
+For example, a new property is available in 9.1.0 for a setting that already existed before 9.1.0.
+
+<image>
+
+% Reference: https://github.com/elastic/kibana/pull/229485/files#r2231856744
+:::{tip}
+If needed, break the contents of the cell into multiple lines using `<br>` to isolate the content you're labeling.
+
+<image>
+:::
```

Comment:
Might also be worth questioning if a table is the right way to display the data. But that's more of a general comment on our docs and not only related to applies_to